src/main/java/com/cupid/jikting/recommend/dto/MemberResponse.java

```diff
@@ -16,11 +24,49 @@ public class MemberResponse {
     private String mbti;
     private String address;
     private String company;
-    private boolean isSmoke;
+    private String smokeStatus;
     private String drinkStatus;
     private int height;
     private String description;
     private List<String> personalities;
     private List<String> hobbies;
     private String college;
+
+    public static MemberResponse toMemberResponse(MemberProfile memberProfile) {
+        return new MemberResponse(
+                memberProfile.getNickname(),
+                getImageResponses(memberProfile),
+                memberProfile.getAge(),
+                memberProfile.getMbti().toString(),
+                memberProfile.getAddress(),
+                memberProfile.getCompany(),
+                memberProfile.getSmokeStatus().getValue(),
+                memberProfile.getDrinkStatus().getValue(),
+                memberProfile.getHeight(),
+                memberProfile.getDescription(),
+                getMemberProfilePersonalities(memberProfile),
+                getHobbies(memberProfile),
+                memberProfile.getCollege());
+    }
+
+    private static List<ImageResponse> getImageResponses(MemberProfile memberProfile) {
+        return memberProfile.getProfileImages().stream()
+                .map(ImageResponse::toImageResponse)
```

Comment:
p2. 체이닝 메소드를 사용할 때는 한 줄에 마침표(.)가 하나씩만 있도록 하는건 어떨까요?

```suggestion
        return memberProfile.getProfileImages()
                .stream()
                .map(ImageResponse::toImageResponse)
```